TWLight/resources/views.py

```diff
@@ -74,10 +74,14 @@ def get_context_data(self, **kwargs):
 
         context['unique_users_approved_or_sent'] = context['unique_users_approved'] + context['unique_users_sent']
 
+        application_end_states = [Application.APPROVED, Application.NOT_APPROVED, Application.SENT]
         partner_app_time = Application.objects.filter(
-            partner=partner).values_list('days_open', flat=True)
+            partner=partner, status__in=application_end_states).exclude(imported=True).values_list('days_open', flat=True)
 
-        context['median_days'] = get_median(list(partner_app_time))
+        if len(partner_app_time) > 0:
+            context['median_days'] = get_median(list(partner_app_time))
+        else:
+            context['median_days'] = "-"
```

Comment:
This is setting this variable to maybe be an int or maybe be a string. The ideal thing to do here would be to set the value to None, rather than setting it to a string. Then the template can have a conditional for if context['median_day'], because None tests false. You could put your dash in the template on that check, or just let it be blank.